cpp/helloworld/Makefile

```diff
@@ -32,7 +32,7 @@
 CXX = g++
 CPPFLAGS = -I/usr/local/include -pthread
 CXXFLAGS = -std=c++11
-LDFLAGS = -L/usr/local/lib -lgrpc -lgrpc++ -lprotobuf -lpthread -ldl
+LDFLAGS = -L/usr/local/lib -lgrpc -lgrpc++_unsecure -lprotobuf -lpthread -ldl
```

Comment:
Reading this, I realize the proper list of libraries should be:
-lgrpc++_unsecure -lgrpc -lgpr

in that order, so that it'd work with statically compiled grpcs.